.teamcity/src/main/kotlin/common/extensions.kt

```diff
@@ -134,7 +134,7 @@ fun BuildType.paramsForBuildToolBuild(buildJvm: Jvm = BuildToolBuildJvm, os: Os)
         }
 
         if (os == Os.MACOS) {
-            param("env.REPO_MIRROR_URLS", "")
```

Comment:
Shall we also remove this as well if we're going to fully adopt `repo.grdev.net`?

Reply:
I'd say yes

cc: @guylabs

Reply:
Yep. I'd give it a try at least to see how it works out.